linalg/cudss.cpp

```diff
@@ -0,0 +1,372 @@
+#include "cudss.hpp"
+#include "../general/communication.hpp"
+#include <string>
+
+#ifdef MFEM_USE_CUDSS
+#ifdef MFEM_USE_MPI
+
+namespace mfem
+{
+namespace
+{
+void CheckCudssAPI(cudssStatus_t status, std::string msg)
```

Comment:
```suggestion
void CheckCudssAPI(cudssStatus_t status, std::string_view msg)
```

(or `const std::string &msh`)

Reply:
Nice solution!